src/app/pages/waiting-room-to-car/waiting-room-to-car.effects.ts

```diff
@@ -67,7 +70,7 @@ export class WaitingRoomToCarEffects {
     // filter any requests that are nulls or empty strings from hitting service
     filter((regNumber) => !!regNumber),
     switchMap((regNumber) => this.vehicleDetailsApiProvider.getVehicleByIdentifier(regNumber)),
-    map((vehicleDetails) => MotStatusChanged(vehicleDetails?.status || MotStatus.NO_DETAILS)),
+    map((vehicleDetails: MotDataWithStatus) => MotStatusChanged(vehicleDetails?.data?.status || MotStatus.NO_DETAILS)),
```

Comment:
Should we still be failing back to `MotStatus.NO_DETAILS`? We are now going to be displaying this value so I would suggest we should not be using a fallback as this will be displayed in search completed